ic-os/components/misc/config.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Shared config utilities.
```

Comment:
Keeping this in one place is a good idea and a nice start.

An ideal (but not realistic right now) is that access to the config structure is also through Rust, so that we can directly track usage for things like, relocating or renaming fields. We can build on this later, though.